apps/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "apps",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@types/node": "20.5.9",
+    "@types/react": "18.2.21",
+    "@types/react-dom": "18.2.7",
+    "autoprefixer": "10.4.15",
+    "eslint": "8.48.0",
+    "eslint-config-next": "13.4.19",
+    "next": "13.4.19",
+    "postcss": "8.4.29",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "tailwindcss": "3.3.3",
```

Comment:
Why do you need tailwindcss?

If you think could be useful for this project, I'd suggest to create an ADR to explain the reason why we introduce tailwind.

[Here is an example of an old ADR](https://github.com/pagopa/developer-portal/blob/main/docs/architecture/decisions/0002-use-nextjs-page-structure.md) written for the developer-portal.
You can start writing a new ADR in this repository, copying the [ADR template](https://raw.githubusercontent.com/pagopa/b2b-portals/main/docs/architecture/decisions/templates/template.md) into the [`decisions` folder ](https://github.com/pagopa/b2b-portals/tree/main/docs/architecture/decisions)